tests/all/char.rs

```diff
@@ -0,0 +1,42 @@
+use super::project;
+
+#[test]
+fn works() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section, wasm_import_module)]
+
+            extern crate wasm_bindgen;
+
+            use wasm_bindgen::prelude::*;
+
+            #[wasm_bindgen]
+            pub fn single_char() -> char { 'a' }
+            #[wasm_bindgen]
+            pub fn wide_char() -> char { '💩' }
+
+            #[wasm_bindgen]
+            pub fn parrot(c: char) -> char { c }
```

Comment:
Could this be expanded to also assert the value of the character coming in to Rust?